lms/djangoapps/discussion/django_comment_client/base/views.py

```diff
@@ -96,10 +99,7 @@ def track_created_event(request, event_name, course, obj, data):
     """
     Send analytics event for a newly created thread, response or comment.
     """
-    if len(obj.body) > TRACKING_MAX_FORUM_BODY:
-        data['truncated'] = True
-    else:
-        data['truncated'] = False
+    data['truncated'] = bool(len(obj.body) > TRACKING_MAX_FORUM_BODY)
```

Comment:
Nit: `bool` isn't necessary (the result of a `>` is already a `bool`)

Reply:
True.  I tried to be as safe as possible and just copied what pylint told me.